www_src/components/option-panel/option-panel.jsx

```diff
@@ -0,0 +1,84 @@
+var React = require('react/addons');
+var classNames = require('classnames');
+
+// Internal -- just the container
+var OptionPanel = React.createClass({
+  render: function () {
+    return (<div className="option-panel">
+      {this.props.children}
+    </div>);
+  }
+});
+
+var Checkbox = React.createClass({
+  mixins: [React.addons.LinkedStateMixin],
+  getDefaultProps: function () {
+    return {
+      linkState: this.linkState,
+      checkedValue: true,
+      uncheckedValue: false
```

Comment:
It sounds like these are labels, not state indicators, in which case we should rename these to `checkedLabel` and `uncheckedLabel` or something, to prevent confusion with the actual checked value that the checkbox yields.